pages/ksphere/kommander/1.1.0-beta/clusters/attach-eks-cluster/index.md

```diff
@@ -132,12 +132,12 @@ This procedure requires the following items and configurations:
 
 For information on related topics or procedures, refer to the following:
 
-- [Configuring and Running Amazon EKS Clusters](https://aws.amazon.com/eks/) 
+- [Configuring and Running Amazon EKS Clusters](https://aws.amazon.com/eks/)
 
-- [Installing and Configuring Konvoy v1.5.0-beta.0 or above](/ksphere/konvoy/1.5.0-beta/install/)
+- [Installing and Configuring Konvoy v1.5.0-beta.0 or above](/ksphere/konvoy/1.5/install/)
```

Comment:
Same here, should it be just v1.5 or nah?
```suggestion
- [Installing and Configuring Konvoy v1.5 or above](/ksphere/konvoy/1.5/install/)
```

Reply:
Good catch, thank you. Will fix.